test/07-openqa-label-known-issues.t

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+source test/init
+bpan:source bashplus +err +fs +sym
+
+plan tests 15
+
+source openqa-label-known-issues
+client_args=(api --host "$host_url")
+
+export KEEP_REPORT_FILE=1
+
+client_output=''
+mock-client-output() {
+    client_output+="client_call $@"$'\n'
+    echo "$client_output\n"
+}
+
+client_call=(mock-client-output "${client_call[@]}")
+echo $dir
+autoinst_log=$dir/data/04-autoinst.txt
+
+try-client-output() {
+    client_output=''
+    try "$*"' && echo "$client_output"'
+}
+openqa-cli() {
+    local id=$(basename "$4")
+    cat "$dir/data/$id.json"
+}
+curl() {
+    if [[ "$7" =~ /(404|414|101|102)/file/autoinst-log.txt ]]; then
+        echo -n "404"
+    else
+        echo -n "200"
+    fi
+}
+comment_on_job() {
+    local id=$1 comment=$2 force_result=${3:-''}
+    echo "$comment"
+}
+
+try investigate_issue
+is "$rc" 1 'id required'
+# error in tap output is from here
+
+issues="159876
+[security] test fails in krb auto_review:\"com:/tmp/nfsdir /tmp/mntdir' failed\":force_result:softfailed
+openqa-force-result
+137420
+[qe-sap] test fails in network_peering auto_review:\"az network vnet peering create.+failed\"
+action
+73375
+Job incompletes with reason auto_review:\"(?m)api failure$\" (and no further details)
+action"
+
+# test data with reason but 404 in curl respond
+id=404
+testurl="https://openqa.opensuse.org/tests/${id}"
+try-client-output investigate_issue $id
+is "$rc" 0 'investigate_issue with missing autoinst-log and with reason in job_data'
+has "$got" "without autoinst-log.txt older than 14 days. Do not label" "exits succefully when is old job without autoinst-log.txt"
+
+# all requisite evidence are missing
```

Comment:
```suggestion
# all assets are missing
```

I assume that's what you meant?

Reply:
i dont think it is accurate but it s fine. done

Reply:
9b5744d